dwisc.py

```diff
@@ -144,12 +178,11 @@ def main(args):
     solutions_all = None
     for i, submitted_problem in enumerate(submitted_problems):
         problem = submitted_problem['problem']
-        await_completion([problem], 1, float('inf'))
+        problem.wait()
```

Comment:
I think here you might want wait_multiple

Reply:
In the dwave_sapi2 lib was I was inclined to only wait on 1 at a time, so that I can provide console feedback on progress.  On the types of inputs we are usually solving, this loop typically takes a few minutes to complete.